rehostimage/chrome/locale/en-US/rehostimage.properties

```diff
@@ -35,6 +35,9 @@ error.downloadfailed=Could not re-download uncached image. Upload failed.
 error.resizefailed=Error resizing image. Upload failed.
 
 error.googleshortenfailed=Shortening URL with goo.gl failed. Using long URL.
+error.isgdshortenfailed=Shortening URL with is.gd failed. Using long URL.
```

Comment:
We should update this to be a format string: Shortening URL with {0} failed. Using long URL.
Then in code insert the actual provider. Or just update to not mention the service used.